layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -304,6 +304,12 @@
   :off (hidden-mode-line-mode)
   :documentation "Toggle the visibility of modeline."
   :evil-leader "tmt")
+(spacemacs|add-toggle display-time
+  :status display-time-mode
+  :on (display-time-mode)
+  :off (display-time-mode -1)
```

Comment:
I didn't know that was available. I had this in my configuration for a while and I don't remember there being a `:mode` option. Updated.